sdk/modelsrepository/Azure.Iot.ModelsRepository/src/ModelPropertyConstants.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Iot.ModelsRepository
+{
+    internal static class ModelPropertyConstants
```

Comment:
class comments for future code owners. What belongs in this class and why?

Reply:
There is a class called ModelRepositoryConstants, we can have these constants in that file as well. even if they are in their own nested subclass.

Reply:
Made it so.